src/CitMovie.Data/DataContext.cs

```diff
@@ -28,5 +29,23 @@ public class DataContext : DbContext
     
     public DataContext(DbContextOptions<DataContext> options) 
         : base(options) { }
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder) {
+        if (!optionsBuilder.IsConfigured)
+        {
+            // var configuration = new ConfigurationBuilder()
+            //     .SetBasePath(Directory.GetCurrentDirectory())
+            //     .AddJsonFile("appsettings.json")
+            //     .AddEnvironmentVariables()
+            //     .Build();
+            //
```

Comment:
Should all this just not be removed if unused?

Reply:
this is going to be used. I should probably uncomment it :D Only it throws error now, but it will work when we merge with Framework.

Reply:
I uncommented it